airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -413,6 +413,7 @@ def _try_reattach_task(self, context):
         )
         if previous_task_arn in running_tasks:
             self.arn = previous_task_arn
+            self.ecs_task_id = self.arn.split("/")[-1]
```

Comment:
Small nit: When I had coded this (before I knew you were working on it :laughing:) I extracted setting the arn/task_id into a helper method, because this code is now duplicated in two spots. Not a huge deal, since ARNs are fairly stable, but worth considering. Approving